src/shared/utils/localStorage.ts

```diff
@@ -1,7 +1,7 @@
 import { LOCALHOST_INPUT_KEY } from "@/shared/constants";
 
-export function getInputFromLocalStorage(key: string): string | null {
-  return localStorage.getItem(key);
+export function getInputFromLocalStorage(): string {
+  return localStorage.getItem(LOCALHOST_INPUT_KEY) ?? "";
 }
 
 export function saveInputToLocalStorage(value: string): void {
```

Comment:
If you have a separate localStorage abstraction, you likely also should have a memory storage which would be used as a mock for tests.